src/App.tsx

```diff
@@ -153,22 +160,34 @@ export const App = () => {
     ]);
   }, [handleCancel, messageHistoryStore, modelConfig.systemPrompt]);
 
+  const isMobile = useMediaQuery('(max-width: 768px)');
```

Comment:
Maybe this could become a hook like useIsMobile so it's easy to remember, import, and keep consistent across the app?

Reply:
thanks i will create a wrapper hook `useIsMobile` that uses this hook underneath